app/client/components/dateInput.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { DateInput, DateInputProps } from "./dateInput";
+
+export default {
+  title: "DateInput",
+  component: DateInput,
+  args: {
+    date: new Date(),
+    labelText: "From",
+    disabled: false,
+  },
+  argTypes: {
+    date: {
+      control: {
+        type: "date",
+      },
+    },
+  },
+} as ComponentMeta<typeof DateInput>;
+
+const Template: ComponentStory<typeof DateInput> = (args: DateInputProps) => {
+  args.date = new Date(args.date);
```

Comment:
The `date` prop expects a `Date` object, but Storybook's date control returns a string.